tests/conftest.py

```diff
@@ -186,3 +192,8 @@ def pytest_configure(config):
 
         print("Setting numba compat :", config.option.relax_numba_compat)
         numba_utils.set_numba_compat_strictness(strict=config.option.relax_numba_compat)
+
+    # Set cache directory for TN/ITN tests
+    from .nemo_text_processing.utils import set_cache_dir
+
+    set_cache_dir(config.option.tn_cache_dir)
```

Comment:
@titu1994 could you please review the changes to the conftest.py?

Reply:
This looks good, I leave final approval to @yzhang123